internal/cmd/testdata/scripts/templatedata.txtar

```diff
@@ -27,8 +27,12 @@ stdout ok
 exec chezmoi execute-template '{{ template "template" . }}'
 stdout ok
 
+-- golden/dot_file --
+dot_file.tmpl
+$WORK/user/.file
```

Comment:
This line needs to be

    $WORK/home/user/.file

Reply:
I added `.chezmoi.templateDir` and so I made it so that this would be just `.file`, which would fix what @bradenhilton saw.